public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -65,33 +65,61 @@ <h5 class="col-xs-12 box-title">
+              maps to Standard Vocabulary {{standardConcepts[0].vocabularyId}} {{standardConcepts[0].conceptCode}}.
+              Standard vocabularies capture data across a variety of source vocabularies.
             </clr-alert>
           </div>
         </div>
         <div class="row">
-          <div class="col-md grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
-          <div class="col-md grid-header flex-xs-middle"> Synonyms </div>
-          <div class="col-md-2 grid-header flex-xs-middle"> Participants of <b> {{totalParticipants | number}}</b> </div>
+          <div class="col-md-3 grid-header flex-xs-middle"> Concept Name</div>
+            <div class="col-md-4 grid-header flex-xs-middle"> Synonyms </div>
+          <div class="col-md-2 grid-header flex-items-xs-right"> Participants of <b> {{totalParticipants | number}}</b> </div>
           <div class="col-md-2 grid-header flex-xs-middle info-text">
             Vocabulary Code
             <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon>
           </div>
           <div class="col-md-1 grid-header flex-xs-middle info-text"> View graphs </div>
-
         </div>
         <div class="row expandable-grid-row"
-             *ngFor="let r of items.concat(standardConcepts), let i = index"
+             *ngFor="let r of standardConcepts"
              [ngClass]="{'expanded-grid-row': r.expanded}" >
-          <div class="col-md flex-xs-middle" innerHtml="{{i+1}}. {{r.conceptName | highlightSearch : prevSearchText}}
+          <div class="col-md-3 flex-xs-middle" innerHtml=" <b>Standard</b> {{r.conceptName | highlightSearch : prevSearchText}}
+            -- {{r.conceptId}}"> </div>
+          <div class="col-md-4 flex-xs-middle" *ngIf="r.conceptSynonyms;else noConceptSynonyms" innerHtml="{{ r.conceptSynonyms.join(', ') | highlightSearch : prevSearchText}}"></div>
+            <ng-template #noConceptSynonyms><div class="col-md-2 flex-xs-middle"></div></ng-template>
+          <div class="col-md-2 flex-items-xs-right" style="margin-top: .5rem;">
+            {{r.countValue | number }}
+          </div>
+          <div class="col-md-2 flex-xs-middle">
+            {{r.vocabularyId}} {{r.conceptCode}}
+          </div>
+          <div class="col-md-1 flex-xs-middle">
+            <button class="icon-btn" (click)="r.expanded = !r.expanded">
+              <clr-icon shape="bar-chart" style="height: 1rem; width: 1rem;color: #2691D0;"></clr-icon>
+            </button>
+          </div>
+          <div class="col-xs-12 row-expansion flex-xs-middle" *ngIf="r.expanded">
+            <app-concept-charts [concept]="r" [showMeasurementGenderBins]="showMeasurementGenderBins" [showAge]="showAge" [showGender]="showGender"
+                                [showSources]="showSources" [backgroundColor]="'#ECF1F4'">
+            </app-concept-charts>
+          </div>
+        </div>
+        <div class="row expandable-grid-row"
```

Comment:
I think we can still do these in just one loop instead of repeating this huge block of code. Instead of  *ngFor="let r of items.concat(standardConcepts), let i = index" 
do 
 *ngFor="let r of standardConcepts.concat(items), let i = index"